test/api/test-aat-layout.c

```diff
@@ -101,13 +101,26 @@ test_aat_get_feature_selectors (void)
   g_assert_cmpuint (0, ==, count);
 }
 
+static void
+test_aat_has (void)
+{
+  hb_face_t *morx = hb_test_open_font_file ("fonts/aat-morx.ttf");
+  g_assert (hb_aat_layout_has_substitution (morx));
+  hb_face_destroy (morx);
+
+  hb_face_t *trak = hb_test_open_font_file ("fonts/aat-trak.ttf");
+  g_assert (hb_aat_layout_has_substitution (trak));
+  hb_face_destroy (trak);
```

Comment:
The test fonts are duplicated of "TestMORXTen.ttf" and "TRAK.ttf" inside the repo